tests/test_util/aiohttp.py

```diff
@@ -62,11 +64,15 @@ def request(
         if params:
             url = url.with_query(params)
 
-        self._mocks.append(
-            AiohttpClientMockResponse(
-                method, url, status, content, cookies, exc, headers
-            )
+        new_response = AiohttpClientMockResponse(
+            method, url, status, content, cookies, exc, headers, long_poll
         )
+        if long_poll and content != b"":
+            for response in self._mocks:
+                if response.long_poll and response.match_request(method, url, params):
+                    response.set_long_poll_response(new_response)
+        else:
+            self._mocks.append(new_response)
```

Comment:
It seems that you always drop the first new_response? (ok from the way you call it, but quite tricky usage to keep in mind)

```suggestion
        if long_poll:
            for response in self._mocks:
                if response.match_request(....):
                    assert response.long_poll
                    response.set_long_poll_response(new_response)
                    return
        self._mocks.append(new_response)
```

Reply:
the issue here is that these long_poll get requests need to be defined before the first http call, but the first response may come later. this is why i defined it so the first call has to be with empty content and sets up the long_poll queue, and then responses are created whenever is right in the flow
perhaps better if i just add asserts that force this behavior (first one must be empty)? this will make it easier to understand.
what i worry about using the first response is that it opens two flows (get request set to just await vs. also enqueu the first command), which can get more confusing
what do you think?

Reply:
added an assert so it will be clear if someone tries the first response with non-empty content